src/codegate/api/v1.py

```diff
@@ -62,9 +62,12 @@ async def create_workspace(request: v1_models.CreateWorkspaceRequest):
     except AlreadyExistsError:
         raise HTTPException(status_code=409, detail="Workspace already exists")
     except ValidationError:
-        raise HTTPException(status_code=400,
-                            detail=("Invalid workspace name. "
-                                    "Please use only alphanumeric characters and dashes"))
+        raise HTTPException(
+            status_code=400,
+            detail=(
+                "Invalid workspace name. " "Please use only alphanumeric characters and dashes"
+            ),
+        )
```

Comment:
can we avoid introducing cosmetic changes or fix our linting job to detect these?